config/util.go

```diff
@@ -40,6 +40,32 @@ func detectConfigFileType(path, def string) string {
 	}
 }
 
+func (b *BgpConfigSet) getPeerGroup(n string) (*PeerGroup, error) {
+	if n == "" {
+		return nil, nil
+	}
+	for _, pg := range b.PeerGroups {
+		if n == pg.Config.PeerGroupName {
+			return &pg, nil
+		}
+	}
+	return nil, fmt.Errorf("no such peer-group: %s", n)
+}
+
+func (d *DynamicNeighbor) Validate(b *BgpConfigSet) error {
```

Comment:
Why exported? Its name is too generic, I think.

Reply:
Sorry, this function is called only from the same package. This should be unexported.
I'll make a fix.